src/input_utils.py

```diff
@@ -311,8 +311,11 @@ def add_side_input_features(
       on_value=1, off_value=0, dtype=tf.int32)
 
   side_inputs = make_relative_transformer_side_inputs(
-      long_breakpoints=features['long_breakpoints'],
+      long_breakpoints=features.pop('long_breakpoints'),
       relative_pos_max_distance=model_config.relative_pos_max_distance)
 
-  features.update(side_inputs.to_dict(exclude_none_values=True))
-  return features
+  features.update(side_inputs.to_dict())
+
+  # TODO (roylu): figure out a better solution
```

Comment:
nit: Make sure comment strings are complete sentences with correct punctuation. This one is missing the "." at the end (and the same for the sentence below, and maybe others).